.buildkite/ftr_security_serverless_configs.yml

```diff
@@ -1,5 +1,7 @@
 disabled:
   # Base config files, only necessary to inform config finding script
+  - x-pack/test_serverless/functional/test_suites/security/cypress/security_config.base.ts
+  - x-pack/test_serverless/functional/test_suites/security/cypress/cypress.config.ts
```

Comment:
@MadameSheema could you confirm this config should be disabled (not run on kibana CI) ?